public/app/plugins/datasource/prometheus/querybuilder/shared/QueryOptionGroup.tsx

```diff
@@ -79,5 +101,12 @@ const getStyles = (theme: GrafanaTheme2) => {
       color: theme.colors.text.secondary,
       marginRight: `${theme.spacing(1)}`,
     }),
+    sizeIndicator: css({
+      margin: '0px',
+      color: theme.colors.text.secondary,
+      // Temporary disabled until we have discussed what the threshold should be
```

Comment:
As mentioned before, I'd remove this code and add it back when we know what treshold are we using.